packages/source/patches/mariadb-connector-c-fix-syntax-error-in-cmake-3.20.diff

```diff
@@ -0,0 +1,11 @@
+--- mariadb.orig/libmariadb/cmake/ConnectorName.cmake	2021-04-14 16:52:11.658995202 +0900
++++ mariadb/libmariadb/cmake/ConnectorName.cmake	2021-04-14 16:52:43.049949830 +0900
+@@ -22,7 +22,7 @@
+     SET(MACHINE_NAME "x64")
+   ELSE()
+     SET(MACHINE_NAME "32")
+-  END()
++  ENDIF()
+ ENDIF()
+ 
+ SET(product_name "mysql-connector-c-${CPACK_PACKAGE_VERSION}-${PLATFORM_NAME}${CONCAT_SIGN}${MACHINE_NAME}")
```

Comment:
Could you use https://github.com/mariadb-corporation/mariadb-connector-c/commit/242cab8cbcd91af882233730a83627d3b12ced83.patch instead of creating a diff manually?

Reply:
Ah...I see.
I'll try it.

Reply:
Can we remove this file?